4-WebApp-your-API/4-2-B2C/TodoListService/Startup.cs

```diff
@@ -46,11 +46,11 @@ public void ConfigureServices(IServiceCollection services)
             {
                 // Create policy to check for the scope 'read'
                 options.AddPolicy("ReadScope", 
-                    policy => policy.Requirements.Add(new OperationAuthorizationRequirement { Name = "read" }));
+                    policy => policy.Requirements.Add(new ScopesRequirement("read")));
             });
 
             // Registering an authorization handler that will check if the scope claim has the requirement specified by the policy
-            services.AddSingleton<IAuthorizationHandler, OperationScopeHandler>();
+            //services.AddSingleton<IAuthorizationHandler, ScopesRequirement>();
```

Comment:
why did you comment this line?

Reply:
I actually need to remove it. Since `ScopesRequirement.cs` extends `AuthorizationHandler`, DI can do this out of the box. This DI configuration is for scenarios where you have a separate class for `AuthorizationHandler`